src/nodule.js

```diff
@@ -44,9 +45,18 @@ export default class Nodule {
             loadFromObject(defaults),
             ...this.loaders,
         );
-        return loader(metadata).then((config) => {
-            bind('config', () => config);
-            return config;
+
+        const declassifiedLoader = loadEach(
+            loadFromObject(defaults),
+            ...this.declassifiedLoaders,
+        );
+
+        return declassifiedLoader(metadata).then((declassifiedConfig) => {
```

Comment:
Why `then` instead of `await`?

Reply:
You have to make it into an async function then and then it gets annoying because then all of the `fromXXX` need to also be async as well.

Reply:
Actually maybe not.. let me try